app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
Code Style: It is recommended to use parentheses `()` when importing multiple modules or classes. This enhances readability and maintainability of your code. Here is how you can do it:

python
from app import (
    Cafe,
    VaccineError,
    NotWearingMaskError
)
```
This way, if you want to add or remove imports in the future, it will result in cleaner diffs in version control systems.